packages/office-ui-fabric-react/src/components/TeachingDialog/TeachingDialogPage.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {
+  ComponentPage,
+  ExampleCard,
+  PropertiesTableSet
+} from '@uifabric/example-app-base';
+
+import { LayerHost } from 'office-ui-fabric-react/lib/Layer';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { TeachingDialogBasicExample } from './examples/TeachingDialog.Basic.Example';
+
+const TeachingDialogBasicExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/TeachingDialog/examples/TeachingDialog.Basic.Example.tsx') as string;
+
+export class TeachingDialogPage extends React.Component<any, any> {
+  public render() {
+    return (
+      <ComponentPage
+        title='TeachingDialog'
+        componentName='TeachingDialogExample'
+        exampleCards={
+          <LayerHost>
+            <ExampleCard title='TeachingDialog' code={ TeachingDialogBasicExampleCode }>
```

Comment:
There is no description, dos/donts, etc. What differentiates this component from a normal dialog?